templates/app/views/products/_products-grid.html.erb

```diff
@@ -10,6 +10,13 @@
 
 <ul class="<%= class_names %>">
   <% products.each do |product| %>
-    <%= render 'products/product-card', product: product %>
+    <%= cache [I18n.locale, current_pricing_options, @taxon, product] do %>
+      <%= render ProductCardComponent.new(
+        product,
+        taxon: @taxon,
+        class_name: local_assigns.fetch(:classes, []).join(" "),
```

Comment:
Does this pass the current partial classes to the inner component?

Reply:
I think so, this is just a way to keep what was there before, specifically on this line:

https://github.com/solidusio/solidus_starter_frontend/blob/7cf7e76d54256cf90e5ab56d1c7805ca84d0a1db/templates/app/views/products/_product-card.html.erb#L8

Although maybe it doesn't make sense in the starter frontend, as nowhere this partial was being called passing an extra `classes` local variable.

Reply:
Correct me if I'm wrong, but I think it's different. 

In the partial, it's using the `local_assigns`, which uses any local variable passed when initializing the partial. But we weren't passing any `classes` local variable when rendering the partial (`<%= render 'products/product-card', product: product %>`). 

With the current code instead, we are passing the local classes to the inner component.

Reply:
Oh good call! Removing it ✂️